R/R/prophet.R

```diff
@@ -177,6 +181,11 @@ validate_inputs <- function(m) {
       validate_column_name(m, h, check_holidays = FALSE)
     }
   }
+  if (!is.null(m$append.holidays)) {
+    if (!m$append.holidays %in% generated_holidays$country){
```

Comment:
According to http://r-pkgs.had.co.nz/data.html  if we set `LazyLoad: true` in the package description it will only be loaded at first use. I also don't think this logical check is going to work because of the operator precedence of `!`.